deploy/k8s/chart/templates/guac/collectsub/030-Deployment.yaml

```diff
@@ -0,0 +1,58 @@
+{{ if .Values.guac.enabled }}
```

Comment:
To me, it's strange to see this condition in a file beneath a directory called "guac". I would rather see this condition in whatever file is referencing this file... I think.